flink/sbt/sbt.rst

```diff
@@ -0,0 +1,57 @@
+===============================
+Using SBT to create a Flink job
+===============================
+
+Motivation
+----------
+
+Maven is a widely accepted build tool for projects that target the Java platform. The Radicalbit Fast Data Distribution provides an archetype to kickstart your development projects using Maven, but Scala developers may find more compelling to use the Maven alternative of choice for the Scala ecosystem, SBT (Simple Build Tool).
+
+Basic 'archetype'
+-----------------
+
```

Comment:
Why not giving user the url of the code? Probably is more simple to checkout that repo and run the provided example. Explanations of steps as below can follow.